interfaces/udev/backend.go

```diff
@@ -69,6 +71,7 @@ func (b *Backend) Setup(snapInfo *snap.Info, opts interfaces.ConfinementOptions,
 		return fmt.Errorf("cannot obtain udev specification for snap %q: %s", snapName, err)
 	}
 	content := b.deriveContent(spec.(*Specification), snapInfo)
+	subsystemTrigger = spec.(*Specification).GetTriggeredSubsystem()
```

Comment:
Didn't you mean to use `:=` assignment? Otherwise the rule mentioned https://github.com/snapcore/snapd/pull/5250/commits/0b7e86f8b3b76faa17148b8e830702c7f0a39ae9#diff-4d456a68388b9a6e3b1a19b9c411816fR127 is no longer true.

Reply:
Actually, it is true because on disconnect nothing is setting it and it defaults to empty. This is only be chance though and you are right, it should be ':='.

Reply:
Done